ansible/roles/jenkins-master/defaults/main.yaml

```diff
@@ -36,3 +36,5 @@ jenkins_plugins:
   - timestamper
```

Comment:
Regarding the commit message, I don't get what you mean. Are you implying that we could keep an encrypted file in the Jenkins node and read/decrypt it when using it in the script?

I agree about using only this script, but the benefit of this will have to weighted against the added time and maintenance effort